fedimint-api/src/core/server.rs

```diff
@@ -203,11 +190,15 @@ where
     }
 
     /// This module's contribution to the next consensus proposal
-    async fn consensus_proposal(&self, dbtx: &mut DatabaseTransaction<'_>) -> Vec<ConsensusItem> {
+    async fn consensus_proposal(
+        &self,
+        dbtx: &mut DatabaseTransaction<'_>,
+        module_instance_id: ModuleInstanceId,
+    ) -> Vec<ConsensusItem> {
         <Self as ServerModulePlugin>::consensus_proposal(self, dbtx)
             .await
             .into_iter()
-            .map(Into::into)
+            .map(|v| (v, module_instance_id).into())
```

Comment:
This `(typed, instance-id).into()` is meh, and would like to make it a standalone function.